calblueprint/unloop
# Divi/fix pundits

Main Changes
- Removes unnecessary views and routes as they did not have sufficient authorization, and are not being used in the app.
- Adds more authorization to the routes that are being used, specifically the root path route
- Removes unnecessary scoping code in pundits.

Note:
I completely gutted the non-api controllers, not sure if this was the best move but I removed the routes and views so we didn't need any of the functions.

To Test:
Try to navigate to all possible routes as a staff or participant, make sure that participants cannot see information they should not be able to see and that I did not break anything.
Most api routes remain untouched.
